tests/grpcio-wheel/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, poetry2nix, python3, pkgs, runCommand }:
```

Comment:
One more thing: what does this test catch that the panel/bokeh one doesn't?

Could we just limit ourselves to one of the two? If not, then maybe adding an explanation of what exactly is being tested could be helpful.

Reply:
Sure, grpcio fails because of a missing wheel on the latest GitHub macos runners.